.drone.yml

```
@@ -7,7 +7,7 @@ pipeline:
       - export DB_HOST='postgres'
       - export AWS_LOCAL_HOST='localstack'
       - export HOCS_DOCUMENT_SERVICE='http://documents:8080'
-      - ./gradlew build --no-daemon
+      - ./gradlew build --no-daemon --info
```

```suggestion
      - ./gradlew build --no-daemon
```

No longer required to have the `--info`

---

Good spot, have removed